Topaz/TopazMain.swift

```diff
@@ -29,7 +29,7 @@ struct TopazMain: App {
 }
 
 private func debouncedJsonFileStorage() -> CodableStorage {
-    DebouncedCodableStorage(JsonDataStorage(FileDataStorage()), debounceInterval: .seconds(2))
+    DebouncedCodableStorage(JsonDataStorage(FileDataStorage()), debounceInterval: .seconds(1))
```

Comment:
Not related but some folks have reported bookmarks not persisting as expected when quickly closing the app when editing so seems prudent to reduce this a bit.